### DISCUSSION TITLE
Chart Dimensions

### DISCUSSION THREAD

**Discussion by takoyaro** [Budibase/budibase#1423] [Ideas]
Currently you can add charts and they're great but it would be even better if we were able to add dimensions to it.

For example, I've got a dataset with daily data-points but I'd like to compare this week's data with last year's data (for the same week). I've been using Datastudio for this purpose until now and I absolutely hate that platform.

**Comment by joebudi** [comment]
Hey @takoyaro 

Thanks for reaching out. it is not possible to add multiple data-points (the same column multiple times) to one chart, over a time period. 

In re to when this available, I am not sure. I will bring it to the next roadmap meeting - but my initial estimate is 4 - 6 months.
